packages/adblocker-benchmarks/blockers/tsurlfilter.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2017-present Cliqz GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+const {
+  Engine,
+  Request,
+  RequestType,
+  RuleStorage,
+  StringRuleList,
+  setConfiguration,
+} = require('@adguard/tsurlfilter/dist/bundle.min.cjs');
```

Comment:
@ameshkov we noticed that minifying code made a huge difference to the initialization time. We are minifying all the main content blockers before running the benchmarks.